README.md

```diff
@@ -1 +1,411 @@
+
+```console
+kubectl create secret generic azure-account-creds -n upbound-system --from-file=credentials=./crossplane-azure-provider-key.json
+```
+
+```console
+cat > provider.yaml <<EOF
+---
+apiVersion: azure.crossplane.io/v1beta1
+kind: ProviderConfig
+metadata:
+  name: default
+  namespace: upbound-system
+spec:
+  credentials:
+    source: Secret
+    secretRef:
+      namespace: upbound-system
+      name: azure-account-creds
+      key: credentials
+EOF
+```
+
+Apply it to the cluster:
+```console
+kubectl apply -f "provider.yaml"
+```
+The output will look like the following:
+
+```shell
+provider.azure.crossplane.io/default created
+```
+
+Crossplane resources use the ProviderConfig named ```default``` if no specific ProviderConfig is specified, so this ProviderConfig will be the default for all Azure resources.
+
+### We are now ready to provision resources:
+
+#### Create Network Fabric
+
+The example network composition includes the creation of a Resource Group, Virtual Network, and Subnets:
+
+```console
+kubectl apply -f examples/network.yaml
+```
+
+Verify status:
+
+```console
+kubectl get claim
+kubectl get composite
+kubectl get managed
+```
+
+#### Invite App Teams to you Organization in Upbound Cloud
+
+1. Create a Team `team1`.
+1. Invite app team members and grant access to `Control Planes` and `Repositories`.
+
+App Dev/Ops: Consume the infrastructure you need using kubectl
+
+#### Join your Organization in Upbound Cloud
+
+1. **Join** your [Upbound Cloud](https://cloud.upbound.io/register)
+   `Organization`
+1. Verify access to your team `Control Planes` and Registries
+
+#### Provision a CompositePostgreSQLInstance in your team Control Plane GUI console
+
+1. Browse the available self-service APIs (XRDs) Control Plane
+1. Provision a `CompositePostgreSQLInstance` using the custom generated GUI for your
+Platform `Configuration`
+1. View status / details in your `Control Plane` GUI console
+
+#### Connect kubectl to your team Control Plane
+
+1. Connect `kubectl` to a `Control Plane` from the self-service GUI console.
+
+#### Provision a PostgreSQLInstance using kubectl
+
+```console
+kubectl apply -f examples/postgres-claim.yaml
+```
+
+Verify status:
+
+```console
+kubectl get claim
+kubectl get composite
+kubectl get managed
+```
+
+### Cleanup & Uninstall
+
+#### Cleanup Resources
+
+Delete resources created through the `Control Plane` Configurations menu:
+
+* From the `Teams` GUI using the ellipsis menu in the resource view.
+* Using `kubectl delete -n team1 <claim-name>`.
+
+Delete resources created using `kubectl`:
+
+```console
+kubectl delete -f examples/postgres-claim.yaml
+kubectl delete -f examples/network.yaml
+```
+
+Verify all underlying resources have been cleanly deleted:
+
+```console
+kubectl get managed
+```
+
+#### Uninstall Provider & Platform Configuration
+
+```console
+kubectl delete configurations.pkg.crossplane.io platform-ref-azure
```

Comment:
these commands didn't work for me, the names of my packages had their org name embedded also, e.g.:

```
kubectl delete configuration.pkg.crossplane.io upbound-platform-ref-azure
kubectl delete provider.pkg.crossplane.io crossplane-provider-azure
kubectl delete provider.pkg.crossplane.io crossplane-provider-helm
```

Reply:
You got me @jbw976 :) I neglected to QA the cleanup pieces. Will run through all of them tomorrow and address.

Reply:
After looking at this, I believe the steps will work in the flow of the readme. At line 143, we pull the package from upbound registry. That instance should match by name to these directions. Later in the readme, we create a package in our personal user org but don't install it. We could add a cleanup to the later directions with `kubectl delete configuration.pkg.crossplane.io/${UPBOUND_ORG}-${REPO}` to clean that up?

Reply:
I think we're talking about the same thing here, this main readme flow of installing platform-ref-azure from the upbound registry, not the personal org flow.  I tried the install/uninstall steps here in the readme to doublecheck this, substituting in platform-ref-gcp instead of azure, since that one is already published and a good example of where this azure one is going, i.e. installed from the upbound cloud registry.

using `kubectl crossplane` v1.5.1 and `up` v0.5.0, I ran:

```
> PLATFORM_VERSION=v0.0.6
> PLATFORM_CONFIG=registry.upbound.io/upbound/platform-ref-gcp:${PLATFORM_VERSION}
> echo $PLATFORM_CONFIG
registry.upbound.io/upbound/platform-ref-gcp:v0.0.6
> kubectl crossplane install configuration ${PLATFORM_CONFIG}
```

resulting in packages:
```
> k get pkg
NAME                                                       INSTALLED   HEALTHY   PACKAGE                                               AGE
configuration.pkg.crossplane.io/upbound-platform-ref-gcp   True        True      registry.upbound.io/upbound/platform-ref-gcp:v0.0.6   3m16s

NAME                                                  INSTALLED   HEALTHY   PACKAGE                                               AGE
provider.pkg.crossplane.io/crossplane-provider-gcp    True        True      registry.upbound.io/crossplane/provider-gcp:v0.19.0   3m12s
provider.pkg.crossplane.io/crossplane-provider-helm   True        True      registry.upbound.io/crossplane/provider-helm:v0.9.0   3m9s
```

Then these deletion instructions result in a "not found" error because they do not match the names of the package objects we are trying to delete:

```
> kubectl delete configurations.pkg.crossplane.io platform-ref-gcp
Error from server (NotFound): configurations.pkg.crossplane.io "platform-ref-gcp" not found
```

Note that the instructions are written are just the package name, but the package objects we should be deleting also has the "org" embedded into the name.  I think for the main readme flow here, these would be the correct deletion instructions:

```
kubectl delete configuration.pkg.crossplane.io upbound-platform-ref-azure
kubectl delete provider.pkg.crossplane.io crossplane-provider-azure
kubectl delete provider.pkg.crossplane.io crossplane-provider-helm
```

Let me know if I don't have something clear from your intent still :)

Reply:
aha, sorry to have you need to go through that in fine detail. I see what you are saying now. Fixed in latest commit.